src/Mod/Path/PathScripts/PostUtilsExport.py

```diff
@@ -228,67 +227,68 @@ def export_common(values, objectslist, filename):
             if values["OUTPUT_COMMENTS"]:
                 if not coolantMode == "None":
                     comment = PostUtilsParse.create_comment(
-                        "Coolant On:" + coolantMode, values["COMMENT_SYMBOL"]
+                        f"Coolant On: {coolantMode}", values["COMMENT_SYMBOL"]
                     )
-                    gcode += PostUtilsParse.linenumber(values) + comment + "\n"
+                    gcode += f"{PostUtilsParse.linenumber(values)}{comment}{nl}"
```

Comment:
```suggestion
            if values["OUTPUT_COMMENTS"] and coolantMode != "None":
                comment = PostUtilsParse.create_comment(
                    f"Coolant On: {coolantMode}", values["COMMENT_SYMBOL"]
                )
                gcode += f"{PostUtilsParse.linenumber(values)}{comment}{nl}"
```

Reply:
Definitely an improvement!  I will make this change.